dbt_platform_helper/utils/platform_config.py

```diff
@@ -1,17 +1,32 @@
-from pathlib import Path
-
 import yaml
 
+from pathlib import Path
+
 from dbt_platform_helper.constants import PLATFORM_CONFIG_FILE
-from dbt_platform_helper.utils.validation import load_and_validate_platform_config
+
+
+def read_config_file():
+    if Path(PLATFORM_CONFIG_FILE).exists():
+        return Path(PLATFORM_CONFIG_FILE).read_text()
+
+
+def load_config_file():
+    file_contents = read_config_file()
+    if not file_contents:
+        return None
+    try:
+        return yaml.safe_load(file_contents)
+    except yaml.parser.ParserError:
+        return None
```

Comment:
I think if you return an empty dict rather than None, there will be less need for error handling.